### DISCUSSION TITLE
Big Query - Bytes column type

### DISCUSSION THREAD

**Discussion by JCruzProbely** [dbeaver/dbeaver#23162] [Q&A] [answered]
### Description

When I perform a SELECT query on a bytes column in BigQuery, the retrieved data appears as something like 'P³ ©¦ ðÀÕâCË' instead of 'GweCULMDqaYQF/DA1eJDyw==', which prompts me to navigate to the BigQuery Console to read the values of this data type.

### DBeaver Version

DBeaver Enterprise Edition

### Operating System

macOS Sonoma 14.3.1

### Database and driver

Host: https://www.googleapis.com/bigquery/v2
Driver Id: google_bigquery_jdbc_simba

**Comment by ShadelessFox** [comment, marked as answer]
Hello @JCruzProbely,

It's just binary data interpreted as a sequence of characters.

If you want to display it as a base64 string, please change its presentation as shown here:
![image](https://github.com/dbeaver/dbeaver/assets/35821147/17d55043-d7c9-46c9-98ee-f1003c37045d)

**Comment by JCruzProbely** [comment]
It works!! 
thank you

**Comment by ShadelessFox** [comment]
I'm closing the ticket as the issue is resolved. Feel free to ask any questions here or via new tickets.
